spec.bs

```diff
@@ -48,134 +50,162 @@ The following example shows a set of sensor values with [=Snippet=]s and afterwa
+  sosa:resultTime "2026-01-01T09:00:00Z"^^xsd:dateTime .
 ```
 </div>
 
 # Definitions # {#definitions}
 
 A <dfn>Time Series</dfn> is a set of data points ordered by timestamp, where each data point consists of a timestamp and corresponding value.
 
-A <dfn>Snippet</dfn> describes a subset of a [=Time Series=] within a certain period defined by a start and end timestamp, and a description of the entity providing the data points.
+A <dfn>Snippet</dfn> describes a subset of a Time Series within a certain period defined by a start and end timestamp, and a description of the entity providing the data points.
 
-A <dfn>Data Point</dfn> is a single point of a [=Time Series=] containing an ISO timestamp, a value with datatype, and optionally an identifier.
+A <dfn>Data Point</dfn> is a single point of a Time Series containing an ISO timestamp, a value with datatype, and optionally an identifier.
 
 # Snippet properties # {#properties}
 
-A [=Snippet=] MUST have the following properties:
+Each Snippet has the following properties:
 
 - `tss:points`: a `cdt:List` of data points where each data point is a `cdt:Map`.
-- `tss:from`: start ISO timestamp (including) of the period covered by `tss:points`.
-- `tss:to`: end ISO timestamp (excluding) of the period covered by `tss:points`.
+- `tss:from`: starting timestamp (including) of the period covered by `tss:points`.
+- `tss:until`: until this timestamp (excluding) of the period covered by `tss:points`.
 - `tss:about`: an entity description which provides the data points in `tss:points`.
 - `tss:pointType`: the RDF type of the data points in `tss:points`.
-- `tss:timePath`: an entity of type `tss:Path` or the term to use for expanding the `time` property in `tss:points`.
-- `tss:valuePath`: an entity of type `tss:Path` or the term to use for expanding the `value` property in `tss:points`.
-
-A [=Snippet=] MAY also have the following properties:
-
-- `tss:identifierPath`: an entity of type `tss:Path` or the term to use for expanding the optional `id` property in `tss:points`.
-
-## Paths ## {#paths}
-
-A [=Snippet=] has multiple path descriptions to define the mapping between CDT and RDF.
-Each of these path descriptions expect an RDF term to use during expansion.
-If nested CDT are used, multiple path descriptions MAY be described, each of type `tss:Path`.
+- `tss:timePath`: the term to use for expanding the `time` property in `tss:points`.
+- `tss:valuePath`: the term to use for expanding the `value` property in `tss:points`.
 
-A `tss:Path` MUST have the following properties:
+Issue: Discuss whether these properties are required or optional. E.g., a publisher might decide to do a lossy conversion for their goal, and not include valuePath, pointType and timePath. However, we can still analyze and visualize the data without that information.
 
-- `tss:term`: the RDF term to use during expansion of the path.
-- `tss:path`: the property name to match with the RDF term.
+## Points ## {#points}
```

Comment:
We should use the same naming convention as the definition we use, this helps readers but also avoid ambiguity.